### ISSUE TITLE
Clean evaluation test for task1

### ISSUE THREAD

**Issue by jplu** [anuzzolese/oke-challenge#29]
Hello,

As promised I pushed a new version of the evaluation dataset for task1.

@anuzzolese: feel free to review and modify it if you see anything that seems wrong to you.

Best.

**Comment by rtroncy** [comment]
Does this new version fixed the issues #8, #24, #25 and #26?

**Comment by jplu** [comment]
No, the issues that you highlight are about the training set, that is not the one I have updated. Anyway I plan as well to update the training set later in order to solve those issues.
